general-superstaq/general_superstaq/check/check_utils.py

```diff
@@ -105,19 +105,39 @@ def get_tracked_files(include: Union[str, Iterable[str]]) -> List[str]:
     """Identify all files tracked by git (in this repo) that match the given match_patterns.
 
     If no patterns are provided, return a list of all tracked files in the repo.
+
+    Args:
+        include: The str of patterns to match files to.
```

Comment:
nit: Should either be
```suggestion
        include: The `str` of patterns to match files to.
```
or 
```suggestion
        include: The string of patterns to match files to.
```
But the preference in the rest of the repo seems to be _string_.